packages/babel-helpers/src/helpers/iterableToArrayLimitLoose.ts

```diff
@@ -0,0 +1,20 @@
+/* @minVersion 7.0.0-beta.0 */
+
+export default function _iterableToArrayLimitLoose<T>(
+  arr: Array<T>,
```

Comment:
```suggestion
  arr: Iterable<T>,
```

Reply:
Ahh, right. Now I recall why I did this. It's because we are accessing the `.length` property below on `arr`, which only exists on `Array`. See: https://github.com/babel/babel/pull/16518#discussion_r1608454438